apps/web/netlify.toml

```diff
@@ -8,10 +8,29 @@
 [context.deploy-preview]
   command = "cd apps/web && pnpm run envsetup && cd ../../ && pnpm run build:web --skip-nx-cache"
 
+[[redirects]]
+  from = "https://dev.web.novu.co/*"
+  to = "https://dev.dashboard.novu.co/:splat"
+  status = 301
+  force = true
+
+[[redirects]]
+  from = "https://eu.web.novu.co/*"
+  to = "https://eu.dashboard.novu.co/:splat"
+  status = 301
+  force = true
+
+[[redirects]]
+  from = "https://web.novu.co/*"
+  to = "https://dashboard.novu.co/:splat"
+  status = 301
+  force = true
+
 [[redirects]]
   from = "/*"
   to = "/index.html"
   status = 200
```

Comment:
Important to keep this redirect last, as Netlify stops evaluating redirects after the first `from` match - keeping this last ensures that the subdomain redirects occur first, stop matching after redirect, then start to match the greedy `/*` redirect lastly.